clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/UpsertSecurityGroupDescription.groovy

```diff
@@ -30,6 +30,8 @@ class UpsertSecurityGroupDescription extends AbstractAmazonCredentialsDescriptio
 
   boolean ingressAppendOnly = false
 
+  Map tags
```

Comment:
Any reason to not use  `com.amazonaws.services.ec2.model.Tag`?

Reply:
```
tags: [
    {
        class: "com.amazonaws.services.ec2.model.Tag",
        value: "0AAB,0AAF",
        key: "nobuObjectIds",
        nobuObjectIds: "0AAB,0AAF"
    },
    {
        class: "com.amazonaws.services.ec2.model.Tag",
        value: "aravinddjavalab-124",
        key: "Name",
        Name: "aravinddjavalab-124"
    }
]
```

Reply:
switched to use `com.amazonaws.services.ec2.model.Tag`

Reply:
Per our discussion offline, we will leave this as a `Map<String, Object>`